Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -3674,12 +3671,8 @@ private List<string> GetWhereClauses(InternalItemsQuery query, IStatement statem
             if (!string.IsNullOrWhiteSpace(nameContains))
             {
                 whereClauses.Add("(CleanName like @NameContains or OriginalTitle like @NameContains)");
-                if (statement is not null)
-                {
-                    nameContains = FixUnicodeChars(nameContains);
-
-                    statement.TryBind("@NameContains", "%" + GetCleanValue(nameContains) + "%");
-                }
+                nameContains = FixUnicodeChars(nameContains);
+                statement?.TryBind("@NameContains", "%" + GetCleanValue(nameContains) + "%");
```

Comment:
Wrap this in a null check again, no need to call `FixUnicodeChars` if we won't use the result